testing/environments/docker/elasticsearch/pki/generate_pki.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+go run ../../../../../dev-tools/cmd/pin_cacert/main.go -cacert ca/ca.crt
```

Comment:
i was looking if we have some ENV which can be useful here instead of ugly `../` like `{ES_BEATS}`maybe?

Reply:
The problem is not available unless you set it up. This is more like an helper methods.

Reply:
I fixed it another way.. I used openssl binaries to generate the pin so the path is now gone.